ui/src/shared/components/LineGraphColorSelector.js

```diff
@@ -0,0 +1,58 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import {connect} from 'react-redux'
+import {bindActionCreators} from 'redux'
+
+import ColorScaleDropdown from 'shared/components/ColorScaleDropdown'
+
+import {updateLineColors} from 'src/dashboards/actions/cellEditorOverlay'
+
+class LineGraphColorSelector extends Component {
+  handleSelectColors = colorScale => {
+    const {handleUpdateLineColors} = this.props
+    const {colors} = colorScale
+
+    handleUpdateLineColors(colors)
+  }
+
+  render() {
+    const {lineColors} = this.props
```

Comment:
this is a bit confusing: `lineColors` is an array? but also expected to be the `selected` one?

Reply:
Since the dropdown is for selecting a group of colors I'm using the cell's line colors in redux as the currently selected colors